desktop/plugins/network/index.tsx

```diff
@@ -370,6 +373,19 @@ export function plugin(client: PluginClient<Events, Methods>) {
         }
         informClientMockChange(routes.get());
       },
+      enableRoute(id: string, routeChange: Partial<Route>) {
```

Comment:
I think you can drop the whole `routeChange` argument, and simplify the entire update to (I found the current implementation a bit hard to follow with the empty object that swaps it's enabled attribute):

```
routes.update((draft) => {
   draft[id].enabled = !draft[id].enabled
});
```

Reply:
I made the change to remove Partial.  However, I'm finding some situations where there is no "id" so I'd like to keep the other code for now (especially since it appears in some of the other existing methods).

Reply:
The other code being:

`if (routes.get().hasOwnProperty(id)) {`